notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -4,7 +4,7 @@
       "cell_type": "code",
```

Comment:
<p>To clean up all Google Cloud resources used in this project, you can&nbsp;<a href="https://cloud.google.com/resource-manager/docs/creating-managing-projects#shutting_down_projects" target="_blank">delete the Google Cloud project</a>&nbsp;that you used for this tutorial.</p><p> </p><p>Otherwise,&nbsp;<strong>if you created the individual resources in the notebook</strong>, you can delete them as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>